controllers/comments.controller.js

```diff
@@ -8,4 +8,15 @@ exports.getComments = (req, res, next) => {
     }).catch((err)=>{
         next(err)
     })
-}
\ No newline at end of file
+}
+
+exports.postComment = (req, res, next) => {
+    const {article_id} = req.params
+    const newComment = req.body
+    
+    addComment(article_id, newComment).then((comment)=>{
+    res.status(201).send({comment})
+    }).catch((err)=>{
+    next(err)
```

Comment:
Here's a smooth express shorthand syntax you can use for the catch:
`.catch(next);` 
That will invoke next with the error 😎